app/views/overview/index.html.erb

```diff
@@ -1,43 +1,45 @@
 <% unless @current_org_signed_mou %>
-<div class="govuk-grid-row">
-  <div class="govuk-warning-text" id="mou-warning">
-    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
-    <strong class="govuk-warning-text__text">
-      <span class="govuk-warning-text__assistive">Warning</span>
-      You must sign the GovWifi <%= link_to "memorandum of understanding", mou_index_path, class: "govuk-link govuk-link--no-visited-state" %> to use this service.
-    </strong>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
```

Comment:
```suggestion
    <div class="govuk-grid-column-full">
```
You're right. But on desktop, it unnecessary wraps (and I remember us fixing this a while back). The suggestion will fix this.

Reply:
> For most types of page, we recommend using either a ‘two-thirds’ or a ‘two-thirds / one third layout’. That stops lines of text getting so long that the page becomes difficult to read on desktop devices.

https://design-system.service.gov.uk/styles/layout/

Reply:
It's already in a three-quarters column, and restricting it further to two-thirds effectively makes it one-half, hence I found it looking a bit odd. This is non-blocking so feel free to disregard if you disagree.